src/state/opamSwitchState.ml

```diff
@@ -476,7 +476,7 @@ let conflicts_with st subset =
 let remove_conflicts st subset pkgs =
   pkgs -- conflicts_with st subset pkgs
 
-let get_conflicts st packages opams_map =
+let get_conflicts package_var packages opams_map =
```

Comment:
Usually, _raw_ functions are named `fname_t` or `fname_raw` (see e.g. in `OpamClient`).It would be better to call this function `get_conflicts_t` (and add it in the mli) and the new one `get_conflicts`.

Reply:
Same as `prepare_package_build` for the `package_var` name.